package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cloudwatch-metrics",
-  "version": "1.2.0",
+  "version": "2.411.0",
```

Comment:
This is a really large bump. Since this doesn't have any external-facing impact, why can't we just move to `1.2.1`? Normally, we prefer making version bumps ourselves - this isn't really part of the PR, but something that comes after.

Reply:
Agree it's a large bump.

1.2.1 doesn't seem to exists on npm, only 1.2.0 which is 6 years old(much older than this project). 

The only reason why i thought it would be fine to bump it, is because it's only using a single api endpoint of cloudwatch that doesn't seem to have changed much. 

This new version allows to only import what's needed from the aws sdk thus reducing the bundle size drastically.

Reply:
I'm confused. This is the version of the `cloudwatch-metrics` module. If you're referring to the `aws-sdk` module, you'll need to bump the version in the `dependencies` hash of `package.json`.

Reply:
arf - my bad :) let me fix this.